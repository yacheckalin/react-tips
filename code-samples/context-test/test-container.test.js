import React from "react";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";

import TestContainer from "./test-container";

describe("<TestContainer />", () => {
  test("displays and shows properly", () => {
    const wrapper = shallow(<TestContainer />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
