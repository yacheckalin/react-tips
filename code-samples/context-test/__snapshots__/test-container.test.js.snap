// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TestContainer /> displays and shows properly 1`] = `
<TestContextProvider>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <TestComponentList />
      <TestComponentButton />
    </div>
  </div>
</TestContextProvider>
`;
